app/controllers/users_controller.rb

```diff
@@ -112,6 +108,17 @@ def reactivate
 
 private
 
+  def valid_attributes?
```

Comment:
Can we rename this? From the name I'd expect this to return a bool and do nothing else, but it's adding errors  to the resource, which isn't clear straightaway

Reply:
Perhaps `validate_attributes`?

Reply:
Changed so it does one thing (validate), we check if we have errors separately